bin/kong-dashboard.js

```diff
@@ -113,6 +120,10 @@ function start(argv) {
     argv.kongRequestOpts.headers['Authorization'] = 'Basic ' + base64;
   }
 
+  if (argv.apiKey !== '') {
+    argv.kongRequestOpts.headers['apikey'] = argv.apiKey;
```

Comment:
"apikey" is the default key name value of the Kong key-auth plugin. But people can change it.

It would be nice to be able to set the key name when starting Kong dashboard, and default to "apikey":
```
kong-dashboard start \
  --kong-url http://kong:8001 \
  --api-key abcdefghijklmnopqrstuvxz1234567890 \
  -- api-key-name my-api-key
```

Reply:
ahh, didn't even realize that, I'm new to Kong, only found it 2 weeks ago :)

will update